clients/src/main/java/org/apache/kafka/clients/consumer/internals/ConsumerCoordinator.java

```diff
@@ -69,6 +70,11 @@
     private final boolean autoCommitEnabled;
     private final AutoCommitTask autoCommitTask;
     private final ConsumerInterceptors interceptors;
+    private final boolean excludeInternalTopics;
+    
+    static {
+        INTERNAL_TOPICS.add("__consumer_offsets");
```

Comment:
One issue with this is that older clients are meant to work with newer brokers and if additional internal topics are added, this could get out of sync.